src/downtimes/downtime_manager.cc

```diff
@@ -489,6 +486,26 @@ int downtime_manager::schedule_downtime(downtime::type type,
   if (start_time >= end_time || end_time <= time(nullptr))
     return ERROR;
 
+  if (start_time > 4102441200) {
+    logger(log_verification_error, basic)
+        << "SCHEDULE DOWNTIME ALERT : start time is out of range and setted to "
+           "4102441200";
+    start_time = 4102441200;
+  }
+
+  if (end_time > 4102441200) {
+    logger(log_verification_error, basic)
+        << "SCHEDULE DOWNTIME ALERT : end time is out of range and setted to "
+           "4102441200";
+    end_time = 4102441200;
+  }
+
+  if (duration > 31622400) {
+    logger(log_verification_error, basic)
+        << "SCHEDULE DOWNTIME ALERT : is too long and setted to 31622400";
```

Comment:
This value has no sense for a human being.
Could we have a duration more readable.